readme.md

```diff
@@ -7,7 +7,7 @@ A Buildkite plugin that allows running additional commands as buildkite hooks on
 ```yaml
 common: &common
   plugins:
-  - improbable-eng/metahook:
+  - improbable-eng/metahook#0.4.1:
```

Comment:
I would be inclined to not include a version here - it's tedious and unnecessary to keep this up to date for reach release

Also this isn't the correct tag, missing 'v'